### DISCUSSION TITLE
Support glob for configuration files

### DISCUSSION THREAD

**Discussion by superdarki** [pelican-dev/panel#608] [Ideas]
Sometimes, there is breaking changes in the path of configurations files and you need to either create two eggs or use a dirty sed in the entrypoint or the startup command.

Supporting globs would mean to me :
  - everything works as usual if no * and no ? are detected in the filename
  - if there is one of theses patterns, match all **existing** files and parse them (it's rare to have two configs needing the same parsing but glob serves the purpose of the changing paths)

**Comment by QuintenQVD0** [comment]
You can parse 2 files in the same configuration parser with the same variables, you just have to do it file by file

@superdarki , there many eggs that do this

**Reply by superdarki** [reply]
That's not the point, but as you made me realize in the PR on wings. Just adding a "doNotCreate" flag or removing the ufs.O_CREATE from the wings file Touch() would do the trick

**Reply by QuintenQVD0** [reply]
It is too late many eggs expect the parser to create the file. an extra flag would be an option (config I guess) but that is not up to me to decide

**Reply by superdarki** [reply]
sure, i'll do my own PR
